src/App.tsx

```diff
@@ -4,14 +4,17 @@ import Footer from "./components/Footer";
 import Navbar from "./components/Navbar";
 import ResultCountLabel from "./components/ResultCountLabel";
 import Tag from "./components/Tag";
+import SearchingSection from "./components/SearchingSection";
 
 const App: FC = () => {
   return (
     <>
       <Navbar />
+      <SearchingSection />
```

Comment:
Or more descriptive? A suggestion:
```suggestion
      <ProblemSearchSection />
```